### ISSUE TITLE
tflocal 0.5: Error: Invalid AWS Region: None

### ISSUE THREAD

**Issue by gh-log** [localstack/terraform-local#7]
Error: Invalid AWS Region: None
   with provider["registry.terraform.io/hashicorp/aws"],
   on main.tf line 11, in provider "aws":
   11: provider "aws" {

Terraform code works fine with AWS and tflocal 0.4.

provider "aws" {
  region = var.aws_region
}

variable "aws_region" {
  description   = "AWS Region"
  type          = string
  default       = "us-east-1"
}

**Comment by lucrohatschIoet** [comment]
I'm having the same issue. 

```
$tflocal --version

Terraform v1.3.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/aws v4.35.0
```

**Comment by mojtaba42** [comment]
I'm running into the same issue

**Comment by HarshCasper** [comment]
Hi @gh-log — Can you please provide me a sample `main.tf` that would allow me to replicate this issue? It would be much appreciated and I can provide a solution.

**Comment by gh-log** [comment]
example code is there.

**Comment by HarshCasper** [comment]
Hi all — I believe that this issue has been fixed with `tflocal 0.6`. Can you please upgrade `tflocal` and try it out once again? Thanks!

**Comment by whummer** [comment]
Closing this issue as resolved - please do let us know if the problem persists.. Thanks!
